ts/components/wallet/payment/PickNotAvailablePaymentMethodListItem.tsx

```diff
@@ -0,0 +1,161 @@
+import { View } from "native-base";
+import * as React from "react";
+import { connect } from "react-redux";
+import * as pot from "italia-ts-commons/lib/pot";
+import { ImageSourcePropType } from "react-native";
+import { GlobalState } from "../../../store/reducers/types";
+import { profileNameSurnameSelector } from "../../../store/reducers/profile";
+import { getFavoriteWalletId } from "../../../store/reducers/wallet/wallets";
+import { PaymentMethod } from "../../../types/pagopa";
+import pagoBancomatLogo from "../../../../img/wallet/cards-icons/pagobancomat.png";
+import bancomatPayLogo from "../../../../img/wallet/payment-methods/bancomatpay-logo.png";
+import satispayLogo from "../../../../img/wallet/cards-icons/satispay.png";
+import { useIOBottomSheet } from "../../../utils/bottomSheet";
+import { H4 } from "../../core/typography/H4";
+import IconFont from "../../ui/IconFont";
+import I18n from "../../../i18n";
+import { IOColors } from "../../core/variables/IOColors";
+import { getCardIconFromBrandLogo } from "../card/Logo";
+import { getBancomatOrCreditCardPickMethodDescription } from "../../../utils/payment";
+import PickPaymentMethodBaseListItem from "./PickPaymentMethodBaseListItem";
+
+type Props = {
+  isFirst: boolean;
+  paymentMethod: PaymentMethod;
+} & ReturnType<typeof mapStateToProps>;
+
+const unacceptedBottomSheetTitle = () =>
+  I18n.t(
+    "wallet.payWith.pickPaymentMethod.notAvailable.unaccepted.bottomSheetTitle"
+  );
+const unacceptedBottomSheetBody = () => (
+  <>
+    <View spacer={true} large={true} />
+    <H4 weight={"Regular"}>
+      {I18n.t(
+        "wallet.payWith.pickPaymentMethod.notAvailable.unaccepted.bottomSheetDescription"
+      )}
+    </H4>
+    <View spacer={true} large={true} />
+  </>
+);
+
+const arrivingBottomSheetTitle = () =>
+  I18n.t(
+    "wallet.payWith.pickPaymentMethod.notAvailable.arriving.bottomSheetTitle"
+  );
+const ArrivingBottomSheetBody = () => (
```

Comment:
```suggestion
const arrivingBottomSheetBody = () => (
```

Reply:
fixed in [a5a4f3f](https://github.com/pagopa/io-app/pull/2926/commits/a5a4f3fea032f1edd2afe786814c52d03e31af5c)